.gitattributes

```diff
@@ -2,4 +2,3 @@
 * text=auto
 
 package.json text eol=lf
-package-lock.json text eol=lf
```

Comment:
Are you sure about this? I've added it because Git will barf every time this file gets changed. I don't know if this is different with yarn.

Reply:
That removal targets a file that gets deleted in this PR.

With regards to `yarn.lock`, I've never had any issue with this eol thingie.

>Git will barf every time this file gets changed.

What kind of barf'ing?

Reply:
> What kind of barf'ing?

The kind that, when you update dependencies, `npm` will save it with `LF`s and, when committing, Git warns you that the file will be restored as `CRLF`. A minor nuisance, for sure.